readme.md

```diff
@@ -38,63 +38,65 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** Forking is a request to clone and registers under your username.
```

Comment:
Not quite. Forking makes a copy of the project on your GitHub account. When you _fork_ a repository, nothing is happening locally. When you `clone` a repository, you're building a connection between a folder on your local machine and what's on GitHub through a remote. When you do both, you get a copy of the project (with all the same code) on both your GitHub account and on your local machine.